index.js

```diff
@@ -544,17 +544,16 @@ Command.prototype.executeSubCommand = function(argv, args, unknown, executableFi
   var localBin = path.join(baseDir, bin);
 
   // whether bin file is a js script with explicit `.js` or `.ts` extension
-  if (exists(localBin + '.js')) {
-    bin = localBin + '.js';
-    isExplicitJS = true;
-  } else if (exists(localBin + '.ts')) {
-    bin = localBin + '.ts';
-    isExplicitJS = true;
-  } else if (exists(localBin + '.mjs')) {
-    bin = localBin + '.mjs';
-    isExplicitJS = true;
-  } else if (exists(localBin)) {
+  if (exists(localBin)) {
     bin = localBin;
+  } else {
+    ['js', 'ts', 'mjs'].some(function(extension) {
```

Comment:
Missing the dots on the file extensions.

Reply:
should I reopen with after having the respective changes :question:

Reply:
No thanks. (I used the review as a quick way of highlighting the broken line of code.)